### ISSUE TITLE
kitti 3d car detection performance

### ISSUE THREAD

**Issue by teddybuy** [traveller59/second.pytorch#37]
How to tweak the hyper-parameters to get closer to the good performance listed in the kitti leaderboard? Thanks!
 
http://www.cvlibs.net/datasets/kitti/eval_object_detail.php?&result=9da5eb9e19aa203a936e2867b44cb24b92f972ac

**Comment by traveller59** [comment]
What's your test performance? This project can directly use pretrained model (car) to generate same labels in that submit. But that model is submitted in 4.28 and the code is changed a lot today. 
The major change is sample functions and noise functions. may be you can tune these parameters.
